perf/tools/setup_env.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+
+# The script setting up a Linux operating system before running
+# Tarantool benchmarks.
+# See https://github.com/tarantool/tarantool/wiki/Benchmarking.
+
+set -eu
+
+uid=$(id -u)
+if [ "$uid" -ne 0 ]
+  then echo "Please run as root."
+  exit 1
+fi
+
+###
+# Helpers.
+###
+
+cpu_vendor="unknown"
+cpuinfo_vendor=$(awk '/vendor_id/{ print $3; exit }' < /proc/cpuinfo)
+if [ "$cpuinfo_vendor" = "GenuineIntel" ]; then
+   cpu_vendor="intel"
+elif [ "$cpuinfo_vendor" = "AuthenticAMD" ]; then
+   cpu_vendor="amd"
+else
+   echo "Unknown CPU vendor '$cpuinfo_vendor'"
+   exit 1
+fi
+
+###
+# Kernel command line parameters.
+###
+
+echo -n "Disable AMD SMT or Intel Hyperthreading "
+sysfs_path="/sys/devices/system/cpu/smt/active"
+if [ -f "$sysfs_path" ]; then
+  is_set=$(cat $sysfs_path)
+  [ "$is_set" = 1 ] && echo "SUCCESS" || echo "FAILED (hint: set 'nosmt' kernel parameter)"
+else
+  echo "SKIPPED"
+fi
+
+echo -n "Isolate CPUs for benchmarking "
+sysfs_path="/sys/devices/system/cpu/isolated"
+if [ -f "$sysfs_path" ]; then
+  isolcpus=$(cat "$sysfs_path")
+  [ -n "$isolcpus" ] && echo "SUCCESS" || echo "FAILED (hint: set 'isolcpus' kernel parameter)"
+else
+  echo "SKIPPED"
+fi
+
+echo -n "Offload interrupts from the isolated CPUs "
+sysfs_path="cat /proc/irq/default_smp_affinity"
```

Comment:
This is not a path, but the content of the file. Thus, the next line is incorrect.
See also [this run](https://github.com/tarantool/tarantool/actions/runs/9911380869/job/27383926994?pr=10219#step:7:44).

Reply:
Right, fixed.
```diff
 echo -n "Offload interrupts from the isolated CPUs "
-sysfs_path="cat /proc/irq/default_smp_affinity"
+sysfs_path="/proc/irq/default_smp_affinity"
 if [ -f "$sysfs_path" ]; then
   smp_affinity=$(cat "$sysfs_path")
-  [ -n "$smp_affinity" ] && echo "SUCCESS" || echo "FAILED (hint: set 'irqaffinity' kernel parameter)"
+  [ -n "$smp_affinity" ] && echo $SUCCESS_MSG || echo "$FAILURE_MSG (hint: set 'irqaffinity' kernel parameter)"
 else
   echo "SKIPPED"
 fi
```